src/components-examples/angular/lightbox/lightbox-component/lightbox-component-example.ts

```diff
@@ -0,0 +1,31 @@
+import { Component } from '@angular/core';
+import { SbbLightbox } from '@sbb-esta/angular/lightbox';
+
+/**
+ * @title Lightbox
+ * @order 20
+ */
+@Component({
+  selector: 'sbb-lightbox-component-example',
+  templateUrl: 'lightbox-component-example.html',
+})
+export class LightboxComponentExample {
+  constructor(public lightbox: SbbLightbox) {}
+
+  openDialog() {
+    const lightboxRef = this.lightbox.open(LightboxComponentExampleContent);
+
+    lightboxRef.afterClosed().subscribe((result) => {
+      console.log(`Lightbox result: ${result}`);
+    });
+  }
+}
+
+/** Lightbox with content loaded from. footer button bar */
+@Component({
+  selector: 'sbb-lightbox-component-example-content',
+  templateUrl: 'lightbox-component-example-content.html',
```

Comment:
I'd inline the template

Reply:
I'd like to extend our showcase example component to allow displaying additional files in the future. And also avoid loading css files that do not exist.